docs/configuring-metabase/config-file.md

```diff
@@ -201,7 +201,15 @@ Some other things to note about API keys in the config file:
 
 ## Referring to environment variables in the `config.yml`
 
-As shown in the Databases examples above, environment variables can be specified with `{% raw %}{{ template-tags }}{% endraw %}` like `{% raw %}{{ env POSTGRES_TEST_DATA_PASSWORD }}{% endraw %}` or `{% raw %}[[options {{template-tags}}]]{% endraw %}`.
+As shown in the examples above, environment variables can be specified with template tags like so:
+
+```
+{% raw %}
+setting: "{{ env POSTGRES_TEST_DATA_PASSWORD }}"
+{% endraw %}
+```
+
+Note the quote marks wrapping the env var template `{% raw %}"{{ env API_KEY_FROM_ENV }}"{% endraw %}`; if you don't include the quotes, the YAML parser won't know it's a string template for Metabase to expand, and Metabase won't know to swap in the env var's value.
```

Comment:
are we good with using "env var" instead of "environment variable"? I can only find one other instance of this usage in the docs